content/v2.0/visualize-data/other-tools/grafana.md

```diff
@@ -11,62 +10,50 @@ weight: 201
 v2.0/tags: [grafana]
 ---
 
-Use the [Grafana Flux Data Source plugin](https://grafana.com/grafana/plugins/grafana-influxdb-flux-datasource)
-to connect Grafana to InfluxDB 2.0 or InfluxDB Cloud 2.0.
+Use [Grafana](https://grafana.com/) to visualize data from **InfluxDB 2.0** and **InfluxDB Cloud**.
```

Comment:
should this use the cloud name shortcode?

Reply:
Good call. Updating.